docs/translations/api-docs-base/dropdown-menu/dropdown-menu.json

```diff
@@ -1,9 +0,0 @@
-{
```

Comment:
Why do we have removed files?

Reply:
The cleaning method was to delete all sub-folders of `docs/translation/api-docs` that are not generated.

Since base and joy are in distinct folder they were not concerned by the cleaning strategy. Those files got removed when I adapted the strategy to work with any folder provided in the setting.

I did not found those file imported anywhere. I assume it's some components name that got modified without removing the auto-generated API